solvers/relay_constellation/mclp_teg_contact_plan/src/solve.py

```
@@ -0,0 +1,360 @@
+    print(f"  Selected: {len(selected)}")
+    print(f"  Baseline score: {mclp_summary.get('baseline_score', 0.0)}")
+    print(f"  Selected score: {mclp_summary.get('selected_score', 0.0)}")
+    print(f"  Scheduler mode: {sched_summary.get('scheduler_mode', 'greedy')}")
+    if sched_summary.get("milp_attempted"):
+        print(f"  MILP attempted: {sched_summary['milp_attempted']}")
+        if sched_summary.get("milp_fallback_reason"):
+            print(f"  MILP fallback reason: {sched_summary['milp_fallback_reason']}")
+    print(f"  Parallel mode: {parallel_mode}")
+    print(f"  Parallel enabled: {use_parallel}")
+    if any_fallback:
+        print(f"  Parallel fallback: yes (sequential used)")
```

Ruff (0.15.11)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)